doc/HACKING/Maintaining.md

```diff
@@ -0,0 +1,115 @@
+# Maintaining Tor
+
+This document details the roles and processes on maintaining the Tor code
+base. We do have many policies in place and the maintainers are there as a
```

Comment:
can we have a link here to the Tor source code that is being mantained? Idem for the policies?

Reply:
They are kind of all in the git tree in `doc/HACKING/`:

- CodingStandards.md and CodingStandardsRust.md
- CodeStructure.md
- HowToReview.md
- WritingTests.md
- GettingStarted.md

Reply:
That last part was removed.